contracts/pl.json

```diff
@@ -0,0 +1,37 @@
+{
+  "generic_goodluck": "Powodzenia!",
+  "generic_objectives": "Aby zawrzeć tę umowę, musisz:",
+  "generic_requirements": "Ten kontrakt pojawi się w grze, jeśli spełnisz następujące wymagania:",
+  "generic_reward": "Za ukończenie tego kontraktu otrzymasz:",
+  "generic_unlocks": "Odblokowuje %name%",
+  "follow_game": "Obserwuj %game_name% na Steam!",
+  "objective_equipment_absolute": "Zbuduj %amount% x %equipment_type% (wliczając obecną konstrukcję przed podpisaniem kontraktu)",
+  "objective_equipment_relative": "Zbuduj %amount% x %equipment_type% (z wyłączeniem tych zbudowanych przed otrzymaniem kontraktu)",
+  "objective_resource_absolute": "Posiadasz %amount% x %resource_type% (w tym bieżące dane uzyskane przed uzyskaniem kontraktu)",
+  "objective_resource_relative": "Posiadasz %amount% x %resource_type% (z wyłączeniem tych uzyskanych przed uzyskaniem kontraktu)",
+  "first_production-title": "Deski rzemieślnicze",
+  "first_production-subtitle": "Zbuduj tartak i wytwarzaj deski.",
+  "first_production-description": "Na początek możesz spróbować pociąć te drewniane kłody na deski i je sprzedać!",
+  "manufacturinc_follow_objective-title": "Obserwuj Manufactur'inc na Steamie",
+  "manufacturinc_follow_objective-subtitle": "Obserwuj stronę gry na Steamie i zdobądź nagrodę!",
+  "manufacturinc_follow_objective-description": "Obserwuj stronę gry na Steamie i zdobądź nagrodę!",
+  "recycle_woodchip-title": "Poddaj wióry drzewne recyklingowi",
+  "recycle_woodchip-subtitle": "Przy ścinaniu drzew powstają wióry. Wykorzystajmy je!",
+  "recycle_woodchip-description": "Podczas ścinania drzew powstają wióry. Wykorzystajmy je do stworzenia czegoś nowego!",
+  "everyone_needs_boxes-title": "Każdy potrzebuje pudełek",
+  "everyone_needs_boxes-subtitle": "Każdy potrzebuje pudełek do każdego celu, więc wyprodukujmy ich trochę!",
```

Comment:
I think we can simplify those two sentences. Can you write "Everyone needs boxes ~~for every purpose~~, so let's produce some!" ?

Reply:
Done!